Automattic/jetpack
# Add/google plus badge widget

Local version of @shield-9's branch in #1604 with additional fixes

replaces #1604
full thanks and props @shield-9

To Test: 
Add widget in both wp-admin and customizer.  Make sure there are no errors when adding either a person, page, or community badge.  

example google+ urls to use: 
- https://plus.google.com/communities/101504763068635549461?hl=en (community)
- https://plus.google.com/+WordPress/posts?hl=en (page)
- https://plus.google.com/u/0/106016569422981142354/posts (profile)
